src/components/html/EcAppsMarket.html

```diff
@@ -1,49 +1,75 @@
 <div class="ec-apps-market">
-  <a-card
-    class="mb-4"
-    :bodyStyle="{ paddingBottom: 0 }"
-  >
-    <h2>{{ i19applications }}</h2>
-    <a-tabs
-      :tabBarStyle="{ margin: 0 }"
+  <div class="card pt-3 pl-3">
+    <h1>
+      {{ i19applications }}
+    </h1>
+    <ul 
+      class="nav nav-tabs" 
       v-model="activeTabKey"
     >
-      <a-tab-pane :tab="i19availableApps" key="market"/>
-      <a-tab-pane :tab="i19yourInstalledApps" key="installed"/>
-    </a-tabs>
-  </a-card>
+      <li 
+        v-for="(label, key) in tabs" 
+        class="nav-item" 
+        :key="key"
+      >
+        <a
+          class="nav-link"
+          :class="{ active: activeTabKey === key }"
+          role="button"
+          @click="activeTabKey = key"
+        >
+          {{ label }}
+        </a>
+      </li>
+    </ul>
+  </div>
 
-  <fade-transition :duration="{ enter: 500, leave: 0 }">
-    <div v-if="!loading" class="pt-2">
-      <a-alert
-        v-if="loadError"
-        key="error"
-        :message="errorMessage"
-        type="warning"
-        closable
-        @close="updateTabContent"
-        :closeText="i19tryAgain"
-      />
+  <div 
+    v-if="!loading" 
+    class="pt-2"
+  >
+    <div 
+      class="card bg-pale-yellow border p-2" 
+      v-if="loadError"
+    >
+      {{ errorMessage }}
+      <div>
+        <a 
+          role="button" 
+          @click="updateTabContent"
+        >
+          {{ i19tryAgain }}
+        </a>
+      </div>
+    </div>
 
-      <article v-else key="list">
-        <div class="row">
-          <div
-            class="col-6 col-md-4 col-lg-3"
-            v-for="(app, i) in apps"
-            :key="`app-${i}`"
-          >
-            <component
-              :is="activeTabKey === 'market' ? 'ec-app-card' : 'ec-installed-app-card'"
-              :app.sync="apps[i]"
-              :ecomApps="ecomApps"
-              @click="$emit('click:app', app)"
-              class="mb-4"
-            />
-          </div>
+    <article 
+      v-else 
+      key="list"
+    >
+      <div class="row">
+        <div 
+          class="col-6 col-md-4 col-lg-3" 
+          v-for="(app, i) in apps"
+          :key="`app-${i}`"
+        >
+          <component
+            :is="activeTabKey === 'market' ? 'ec-app-card' : 'ec-installed-app-card'"
+            :app.sync="apps[i]"
+            :ecomApps="ecomApps"
+            @click="$emit('click:app', app)"
+            class="mb-4"
+          />
         </div>
-      </article>
+      </div>
+    </article>
+  </div>
+  <div 
+    v-if="loading"
+    class="d-flex justify-content-center"
+  >
+    <div class="spinner-border" role="status">
+      <span class="sr-only">Loading...</span>
```

Comment:
Agora funciona (:smile: ) mas acho que esse spinner aqui tá pequeno demais, não? Acho que aqui ele está ficando com o tamanho padrão mesmo, e considerando que ele é o único conteúdo da view neste momento acho que ele deveria ocupar um espaço maior...

Reply:
Você pode setar width e height dele com inline style mesmo se for o caso...